doc/manuals/user/ngsiv2_implementation_notes.md

```diff
@@ -238,6 +238,8 @@ The particular validations that Orion implements on NGSIv2 subscription payloads
 * **throttling**: optional (must be an integer)
 * **expires**: optional (must be a date or empty string "")
 * **status**: optional (must be a valid status keyword)
+* **Orion extension**:
```

Comment:
Maybe I didn't explain clearly enough...

What I mean is to create a new section in this .md document, I mean, this way:

```
## Additional fields in `httpInfo` for custom notifications
```

and inside that section explain the new field `includePayload`.

The section `## Subscription payload validations` doesn't need any change. Please, rollback your addition there so it looks as before.

Reply:
Updated